tests/scripts/check_test_cases.py

```diff
@@ -99,17 +101,11 @@ def walk_test_suite(self, data_file_name):
     def walk_ssl_opt_sh(self, file_name):
```

Comment:
Since `ssl-opt.sh` now takes care of looking at the files in the `opt-testcases` subdirectory, I think you'll want to revert f17a60f147ec3ba6a9f76da38d3802a7f9f05227.

Reply:
Done